### ISSUE TITLE
Issue with installing autorest@latest

### ISSUE THREAD

**Issue by alexhcheng** [Azure/autorest#3899]
```
npm remove -g autorest
npm install -g autorest
> autorest@3.1.0 preinstall C:\Users\alcheng\AppData\Roaming\npm\node_modules\autorest
> node ./.scripts/preinstall-check

C:\Users\alcheng\AppData\Roaming\npm\autorest -> C:\Users\alcheng\AppData\Roaming\npm\node_modules\autorest\entrypoints\app.js
+ autorest@3.1.0
added 1 package from 1 contributor in 0.793s

autorest --help
AutoRest code generation utility [cli version: 3.1.0; node: v14.15.5, max-memory: 4096 MB]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
Failure:
Error: Unable to find a valid AutoRest core package '@autorest/core' @ '~3.1.0'.
Error: Unable to find a valid AutoRest core package '@autorest/core' @ '~3.1.0'.
    at Object.selectVersion (C:\Users\alcheng\AppData\Roaming\npm\node_modules\autorest\dist\src_autorest-as-a-service_ts-src_coloring_ts.js:297:23)
    at async main (C:\Users\alcheng\AppData\Roaming\npm\node_modules\autorest\dist\app.js:213:31)
```

**Comment by alexhcheng** [comment]
manually installing from one of the releases works fine
```
npm install -g autorest@3.0.6334
autorest --help
AutoRest code generation utility [cli version: 3.0.6334; node: v14.15.5, max-memory: 4096 MB]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest

There is a new version of AutoRest available (3.1.0).
 > You can install the newer version with with npm install -g autorest@latest

   Loading AutoRest core      'C:\Users\alcheng\.autorest\@autorest_core@3.0.6374\node_modules\@autorest\core\dist' (3.0.6374)


Usage: autorest [configuration-file.md] [...options]
...
```

**Comment by ImoutoChan** [comment]
Have the same issue here

**Comment by timotheeguerin** [comment]
Published a fix, please update `autorest` to `autorest@3.1.1`
